app/src/main/java/dev/aldi/sayuti/block/ExtraPaletteBlock.java

```diff
@@ -84,19 +83,10 @@ private boolean isWidgetUsed(String str) {
         return false;
     }
 
-    public void f(Ss ss) {
-        String menuName = ss.getMenuName();
-        if (ss.b.equals("m")) {
-            if (menuName.matches("cardview|collapsingtoolbar|textinputlayout|swiperefreshlayout|radiogroup|lottie|otpview|youtubeview|codeview|recyclerview|datepicker|timepicker")) {
-                logicEditor.f(ss);
-            } else if (menuName.matches("Assets|NativeLib")) {
-                new ExtraMenuBean(logicEditor).defineMenuSelector(ss);
-            } else {
-                logicEditor.g(ss);
-            }
-        }
-    }
-
+    /**
+    * ExtraPaletteBlock#f(Ss) moved to mod.w3wide.menu.ExtraMenuBean#defineMenuSelector(Ss)
+    * for better block menu selections and to add new stuff easily.
+    */
```

Comment:
Why is this a Javadoc of `ExtraPaletteBlock#e` though?

Reply:
Actually, this javadoc is not for `ExtraPaletteBlock#e`, it's for `ExtraPaletteBlock#f`.